src/Lib/Configuration/ConfigurationFactory.php

```diff
@@ -9,20 +11,22 @@ final class ConfigurationFactory
 {
 
     /** @var string */
-    private const DIRECTORY = __DIR__ . "/../../../config/";
+    private const CONFIG_DIRECTORY = __DIR__ . "/../../../config/";
 
     /**
      * Creates configuration from filename
      *
      * @param string $fileName
      * @return Configuration
      */
-    public static function fromFileName(string $fileName):Configuration
+    public static function fromFileName(string $fileName): Configuration
     {
-        if (empty($fileName) || !($path = realpath(self::DIRECTORY . $fileName . '.ini'))) {
-            throw new Exception('Config file ' . $fileName . ' not exists!');
+        $configs = glob(self::CONFIG_DIRECTORY . $fileName . '*' . 'ini');
```

Comment:
missing a dot?
`$fileName . '*' . '.ini');`

non the less i have no idea why there sould be '*', when you pass the file name somewhere else

Reply:
True, fixed!